.gitignore

```diff
@@ -16,6 +16,7 @@ muter.xcodeproj
 muterReport.json
 muter_logs
 AcceptanceTests/samples/*.*
+AcceptanceTests/Repositories/
```

Comment:
do we have this directory or am I missing somehting?

Reply:
I modified the **runAcceptanceTests.sh** script to copy the Repositories folder into the AcceptanceTests directory. Thus, AcceptanceTests/Repositories should be ignored. The script removes AcceptanceTests/Repositories every time it is run.

Reply:
If run the Muter on the current Repostiroies folder and something goes wrong, the temporary folders will remain. This will not allow you from running the test again and you will need to delete them manually.

Reply:
that makes sense, thanks!